### ISSUE TITLE
add additional metrics like beta, covariance, seasonalilty

### ISSUE THREAD

**Issue by camelbyte** [camelbyte/stock-prediction#2]
https://github.com/camelbyte/stock-prediction/blame/2ad8337cd0a9e6ba40be077cf88e4877fe4d15dc/train-test.ipynb#L374C3-L374C3

**Comment by camelbyte** [comment]
df.set_index('Date', inplace=True)\n",
    "features = df[['Open', 'High', 'Low', 'Close', 'Volume']]\n",
    "target = df['Close']\n",
    "X_train, X_test, y_train, y_test = train_test_split(features, target, test_size=0.2, random_state=42)\n",
    "\n",
    "scaler = StandardScaler()\n",
    "X_train_scaled = scaler.fit_transform(X_train)\n",
    "X_test_scaled = scaler.transform(X_test)\n",
    "\n",
    "model = LinearRegression()\n",
    "model.fit(X_train_scaled, y_train)\n",
    "\n",
    "y_pred = model.predict(X_test_scaled)\n",
    "\n",
    "mse = mean_squared_error(y_test, y_pred)\n",
    "r2 = r2_score(y_test, y_pred)\n",
    "\n",
    "print(f'Mean Squared Error: {mse}')\n",
    "print(f'R^2 Score: {r2}')\n",
    "\n",
    "test_dates = df.loc[X_test.index].index\n",
    "\n",
    "plt.figure(figsize=(10, 6))\n",
    "plt.plot(test_dates, y_test.values, label='Actual Prices')\n",
    "plt.plot(test_dates, y_pred, label='Predicted Prices', linestyle='--')\n",
    "plt.xticks(rotation=60, fontsize=6)\n",
    "plt.xlabel('Date')\n",
    "plt.ylabel('Price')\n",
    "plt.title('AAPL Stock Price Prediction')\n",
    "plt.legend()\n",
